keras/callbacks.py

```diff
@@ -580,9 +584,13 @@ def on_epoch_end(self, epoch, logs=None):
             else:
                 send[k] = v
         try:
-            requests.post(self.root + self.path,
-                          {self.field: json.dumps(send)},
-                          headers=self.headers)
+            if self.send_as_json:
+                payload = {self.field: send} if self.field else send
```

Comment:
this if is not documented

Reply:
@Dref360  There is this fragment in the class documentation:
<code>If send_as_json is set to True, the content type of the request will be application/json.
Otherwise the serialized JSON will be send within a form</code>

Do you want me to add something more?

Reply:
`field` can be `None` and it would also work. But it's not documented in the `field` doc

Reply:
It's also inconsistent with line 593

Reply:
@Dref360 the <code>field</code> param makes sense only when the data is sent within the form (the old path), there is no necessity to wrap the data when it is sent as <code>application/json</code>, so I've now simplified the logic and updated the docs.